ipyleaflet/leaflet.py

```diff
@@ -881,6 +892,15 @@ def __init__(self, **kwargs):
         super(Map, self).__init__(**kwargs)
         self.on_displayed(self._fire_children_displayed)
         self.on_msg(self._handle_leaflet_event)
+        
+    @observe('zoom_control')
+    def observe_zoom_control(self, change):
+        if change['new']:
+            self.add_control(ZoomControl())
+        else:
+            for control in self.controls:
+                if isinstance(control, ZoomControl):
+                    self.remove_control(control)   
```

Comment:
There might be a cleaner way to do it, by storing the ZoomControl instance as a Map attribute.

Anyway, I don't want to block your PR for that. We can clean it later :)

Reply:
i can do the modification now.